src/utils/createFeatureStoreContent.js

```diff
@@ -75,7 +75,9 @@ const createFeatureSetsRowData = (featureSet, project) => {
       type: 'hidden'
     },
     entity: {
-      value: featureSet.entities ? featureSet.entities[0]?.name : '',
+      value: featureSet.entities[1]?.name
+        ? `${featureSet.entities[0]?.name} ${featureSet.entities[1]?.name}`
```

Comment:
instead of ternary operator, use nullish coalescing operator to check if we have a second entitie